### ISSUE TITLE
Compress and bundle CSS before release

### ISSUE THREAD

**Issue by davecranwell** [wagtail/wagtail#1448]
Our gulpfile currently only processes SASS with the "Extended" output style which leaves in all our comments and doesn't compress it at all. (https://github.com/sass/node-sass#outputstyle)

Our vanilla CSS similarly is not processed, and remains at practically full size.

We should set gulp-sass to use "compressed" output style and install webpack/browserify/whatevs to bundle CSS better. This would be a precursor to dropping the django-compressor dependency completely, once JS had been similarly bundled by external scripts, but that's for another Issue.

**Comment by lee-kagiso** [comment]
I tried pinning to master to get all the latest fixes etc. and I get this error

```
Template error:
In template /home/lee/Code/cms/venv/lib/python3.4/site-packages/wagtail/wagtailadmin/templates/wagtailadmin/skeleton.html, error at line 13
   'wagtailadmin/css/normalize.css' could not be found in the COMPRESS_ROOT '/home/lee/Code/cms/staticfiles' or with staticfiles.```

I assume this is related?

**Comment by davecranwell** [comment]
@lee-kagiso that error occurs for anyone using master who hasn't yet compiled the CSS themselves. Only the packaged version of Wagtail includes compiled CSS. Anyone daring enough to check out a version from github is expected to compiled the CSS themselves using `npm build` wherever wagtail is installed. See: http://docs.wagtail.io/en/latest/contributing/developing.html#compiling-static-assets

**Comment by thibaudcolas** [comment]
Note: this is related to #2189. I'll most likely close this once #2189 is merged.

**Comment by jjanssen** [comment]
Merged with a10e1f0
